src/components/Auth/SignUp/SignUp.css

```diff
@@ -127,6 +127,19 @@
     color: #fff;
 }
 
+.is-strength-0 { color: #D1462F; }
+.is-strength-1 { color: #D1462F; }
+.is-strength-2 { color: #57B8FF; }
+.is-strength-3 { color: #57B8FF; }
+.is-strength-4 { color: #2FBF71; }
+
+.is-strength-name-0 { margin-left: 44%;}
```

Comment:
Instead of using variable margin for each strength, use margin-auto and align it center to the container

Reply:
These changes are unattended.

Reply:
Make margin as auto